### PR TITLE
tests/many: enable some uc20 tests, delete old unneeded tests or TODOs

### PR DESCRIPTION
**Pull request by anonymouse64**
Adjust various spread tests to be enabled on UC20.

I didn't run through all of these tests, but some of the ones I did run through passed when run manually in a UC20 VM.

The netplan test we should really fix at some point since that is probably an important commercial feature, but it does work fine IRL when run outside of spread, just the test setup is wrong/broken. We actually may have other such issues in our tests given that the classic and core gid/uids don't always match up like this but I don't know for sure.

### CODE REVIEW

**Comment by anonymouse64** [comment]
Seems all the uc20 tests passed here, but many others did not :-( I will look into this, please don't restart the tests

**Review by bboozzoo — APPROVED** [review]
LGTM

File: tests/core/uboot-unpacked-assets/task.yaml [hunk 1]
```diff
@@ -1,15 +1,23 @@
...
     NAME/kernelimg: kernel.img*
 
 execute: |
-    output=$(find /boot/uboot/*-kernel_*.snap/ -name "$NAME" )
-    if [ -z "$output" ]; then
-        echo "Not found expected file $NAME in /boot/uboot/*-kernel_*.snap/"
-        exit 1
+    if os.query is-core20; then
+        # on UC20, the kernel snap is extracted onto ubuntu-seed, not on ubuntu-boot
+        output=$(find /run/mnt/ubuntu-seed/systems/*/kernel/*-kernel_*.snap/ -name "$NAME" )
```
Comment (sergiocazzolato) [inline tests/core/uboot-unpacked-assets/task.yaml:12]: is this path correct for arm devices? Could be on /run/mnt/ubuntu-seed/snaps instead?

Comment (anonymouse64) [reply tests/core/uboot-unpacked-assets/task.yaml:12]: There are kernel snap files under /snaps but the kernel snap is not extracted there

Comment (sergiocazzolato) [reply tests/core/uboot-unpacked-assets/task.yaml:12]: ah, ok, thanks for the answer

**Review by sergiocazzolato — APPROVED** [review]
Thanks

**Comment by anonymouse64** [comment]
I had a look at the tests here and they're not an issue, so this can go in when it's green or if someone wants to force merge this.

**Pull request merged**